src/features/form/Form.tsx

```diff
@@ -70,6 +73,14 @@ export function PostForm({
 				  }
 	})
 
+	if (userNamesMap.size === 0) {
+		fetchUsers([]).then(users => {
+			for (const user of users) {
+				setUserNamesMap(usersMap => new Map(usersMap).set(user.name, user.id))
+			}
```

Comment:
全ユーザー分でsetすると遅いので、こうするといいと思います
```suggestion
			setUserNamesMap(new Map(users.map(user => [user.id, user.name])))
```